curricula/templates/curricula/partials/admin_menu.html

```diff
@@ -8,45 +8,62 @@
             <h2 class="drawer-title">Settings</h2>
         </div>
         <div class="drawer-body">
+
             <h2>Add to Log</h2>
             <form method="POST" action="." class="post-form">{% csrf_token %}
                 {{ form.as_p }}
                 <button type="submit" class="save btn btn-primary">Save</button>
             </form>
-            <h3><a href="/page_history/{{ lesson.pk }}/">View history</a></h3>
+            {% if unit %}
+            <h3><a href="unit_feedback/">View unit feedback</a></h3>
```

Comment:
nit: can we try to use the `{% url ... %}` tag as much as possible for things like this?

Reply:
updated